ts/src/coinlist.ts

```diff
@@ -0,0 +1,2246 @@
+            'quoteVolume': quoteVolume,
+            'info': ticker,
+        }, market);
+    }
+
+    async fetchOrderBook (symbol: string, limit: Int = undefined, params = {}) {
+        /**
+         * @method
+         * @name coinlist#fetchOrderBook
+         * @description fetches information on open orders with bid (buy) and ask (sell) prices, volumes and other data
+         * @param {string} symbol unified symbol of the market to fetch the order book for
+         * @param {int} [limit] the maximum amount of order book entries to return (default 100, max 200)
+         * @param {object} [params] extra parameters specific to the coinlist api endpoint
+         * @returns {object} A dictionary of [order book structures]{@link https://github.com/ccxt/ccxt/wiki/Manual#order-book-structure} indexed by market symbols
+         */
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'symbol': market['id'],
+        };
+        const response = await this.publicGetV1SymbolsSymbolBook (this.extend (request, params));
+        //
+        //     {
+        //         "bids": [
+        //             [ "30900.00000000", "0.0001" ],
+        //             [ "30664.21000000", "0.0172" ],
+        //             [ "30664.20000000", "0.0906" ],
+        //         ],
+        //         "asks": [
+        //             [ "31349.99000000", "0.0003" ],
+        //             [ "31350.00000000", "0.0023" ],
+        //             [ "31359.33000000", "0.0583" ],
+        //         ],
+        //         "after_auction_code": "BTC-USDT-2023-10-23T18:40:51.000Z",
+        //         "call_time": "2023-10-23T18:40:51.068Z",
+        //         "logical_time": "2023-10-23T18:40:51.000Z"
+        //     }
+        //
+        const logical_time = this.parse8601 (this.safeString (response, 'logical_time'));
+        const call_time = this.parse8601 (this.safeString (response, 'call_time'));
+        const orderbook = this.parseOrderBook (response, symbol, logical_time);
+        orderbook['nonce'] = call_time;
+        return orderbook;
+    }
+
+    async fetchOHLCV (symbol: string, timeframe = '1m', since: Int = undefined, limit: Int = undefined, params = {}) {
+        /**
+         * @method
+         * @name coinlist#fetchOHLCV
+         * @description fetches historical candlestick data containing the open, high, low, and close price, and the volume of a market
+         * @param {string} symbol unified symbol of the market to fetch OHLCV data for
+         * @param {string} timeframe the length of time each candle represents
+         * @param {int} [since] timestamp in ms of the earliest candle to fetch
+         * @param {int} [limit] the maximum amount of candles to fetch
+         * @param {object} [params] extra parameters specific to the coinlist api endpoint
+         * @returns {int[][]} A list of candles ordered as timestamp, open, high, low, close, volume
```

Comment:
we can unify the `until` parameter since the exchange supports it. 
Basically you can add this to the docs

```
         * @param {int} [params.until] the latest time in ms to fetch entries for
```
and then do:

```Javascript
        const until = this.safeInteger2 (params, 'till', 'until');
        if (until !== undefined) {
            params = this.omit (params, [ 'till', 'until' ]);
            request['end_time'] = this.iso8601 (until);
        }
```

⬆️  this applies to every method that supports `end_time`